tests/testthat/test_calc_auc_analytic.R

```diff
@@ -0,0 +1,114 @@
+## Create reference data:
+reg <- new_regimen(amt = 1500, n = 10, interval = 24, type = "infusion")
+run_ref <- FALSE
+if(run_ref) {
+  library(pkvancothomson)
+  par <- pkvancothomson::parameters()
+  mod <- pkvancothomson::model()
+  covs <- list(
+    WT = new_covariate(70),
+    CRCL = new_covariate(4),
+    CL_HEMO = new_covariate(0)
+  )
+  t_obs <- seq(0, 24*10, 24)
+  res <- sim(
+    mod,
+    parameters = par,
+    regimen = reg,
+    covariates = covs,
+    t_obs = t_obs
+  ) %>%
+    dplyr::filter(comp == attr(mod, "size")) %>%
+    dplyr::mutate(auc = round(c(0, diff(y)), 3)) %>%
+    select(t, auc)
```

Comment:
```suggestion
  res <- sim(
    mod,
    parameters = par,
    regimen = reg,
    covariates = covs,
    t_obs = t_obs
  ) %>%
  res <- res[res$comp == attr(mod, "size"))
  res$auc <- round(c(0, diff(res$y)), 3)
  res <- res[, c("t", "auc")]
```

This is causing an issue with R CMD check.